### ISSUE TITLE
Live audio transcription / translation

### ISSUE THREAD

**Issue by ahcm** [bigbluebutton/bigbluebutton#17396]
Hi,

using something like whisper.cpp, https://github.com/ggerganov/whisper.cpp, could enable live audio transcription and translation.

Is someone already working on something like this?

Thanks!
Andreas

**Comment by hiroshisuga** [comment]
This seems planned for BBB2.7: #16442
Whisper looks wonderful!

**Comment by ffdixon** [comment]
Yes.  You'll see #16442 merge into 2.7 development and form the basis for work on incorporating Whisper as well.

**Comment by JulianKropp** [comment]
Hey. I want to let you know I started a new project a year ago. Instead of adding a new feature to BBB, I started to develop a bot that joins the audio channel listens to the conversation, and starts converting audio into text. It can also translate many languages into English. Everything works completely offline and without any cloud like Google or Microsoft. I'm using the open-source AI whisper from Open AI.

Have a look and try it out. I would love to get some feedback and some support to continue developing my Big Blue Button translation bot.

[bbb-translation-bot](https://github.com/ITLab-CC/bbb-translation-bot)

[bbb-bot](https://github.com/ITLab-CC/bigbluebutton-bot)
